ceph.spec.in

```diff
@@ -64,6 +64,13 @@ Group:         System/Filesystems
 %endif
 URL:		http://ceph.com/
 Source0:	http://ceph.com/download/%{name}-%{version}.tar.bz2
+%if 0%{?suse_version}
+%if 0%{?is_opensuse}
+ExclusiveArch x86_64 aarch64 ppc64 ppc64le
+%else
+ExclusiveArch x86_64 aarch64
```

Comment:
I removed those changes as not related to 
http://tracker.ceph.com/issues/16937